application/helper/returnallsuites.py

```diff
@@ -3,7 +3,7 @@
 from application.model.models import TestSuite
 
 
-def return_all_suites(user_id):
+def return_all_suites(project_id):
     """
     Method that returns all the suite, case details associated with the user.
     Args:
```

Comment:
docstring is not proper, describe what project_id is for and returns what

Reply:
For now, we are not validating this check. We will add it later.